ServerCore/Pages/Teams/TeamListBase.cs

```diff
@@ -24,18 +24,32 @@ public TeamListBase(PuzzleServerContext serverContext, UserManager<IdentityUser>
         /// </summary>
         public int PlayerCount { get; set; }
 
-        protected async Task LoadTeamDataAsync()
+        protected async Task LoadTeamDataAsync(bool onlyIncludeTeamsLookingForMembers)
         {
-            List<Team> allTeams = await (from team in _context.Teams
-                                         where team.Event == Event
-                                         select team).ToListAsync();
-
-            Teams = await (from team in _context.Teams
-                           where team.Event == Event
-                           join teamMember in _context.TeamMembers on team equals teamMember.Team
-                           group teamMember by teamMember.Team into teamCounts
-                           select new { Team = teamCounts.Key, Count = teamCounts.Count() }).ToDictionaryAsync(x => x.Team, x => x.Count);
-
+            List<Team> allTeams = null;
+            if (onlyIncludeTeamsLookingForMembers) {
```

Comment:
I don't like this part because it skews the registration counts for teams and players. I'd rather we just remove teams from the list at the end, after that's been properly calculated.

Reply:
Moved the check to the list html file - thanks for helping debug!